charts/consul/values.yaml

```diff
@@ -263,6 +263,46 @@ global:
         additionalConfig: |
           {}
 
+      controller:
+        # The directory that Kubernetes will use on Kubernetes CRD creation, 
+        # deletion, and update, to get TLS certificates used issued from vault
+        # to send webhooks to the controller.
+        tlsCert:
+          # @type: string
+          secretName: null
+          
+        # The directory that Kubernetes will use on Kubernetes CRD creation, 
+        # deletion, and update, to get CA certificates used issued from vault
+        # to send webhooks to the controller.
+        caCert:
+          # @type: string
+          secretName: null
+      
+      # The Vault role for consul controller to read the Consul's controllers's CA Certificate (unauthenticated).
+      # A Vault policy must be created which grants read capabilities to 
+      # `global.secretsBackend.vault.controller.tlsCert.secretName`.
+      consulControllerCARole: ""
```

Comment:
I'd prefer to put this role inside of the `controller` stanza above and name it something more succinct.  I left this outside of that stanza because all of the previously defined roles for like `consulCARole`,`adminPartitionsRole`, manageSystemACLsRole`, etc are at this level.

Same for `consulConnectInjectCARole` below.